adafruit_bitbangio.py

```diff
@@ -37,36 +45,36 @@
 class _BitBangIO:
     """Base class for subclassing only"""
 
-    def __init__(self):
+    def __init__(self) -> None:
         self._locked = False
 
-    def try_lock(self):
+    def try_lock(self) -> bool:
         """Attempt to grab the lock. Return True on success, False if the lock is already taken."""
         if self._locked:
             return False
         self._locked = True
         return True
 
-    def unlock(self):
+    def unlock(self) -> None:
         """Release the lock so others may use the resource."""
         if self._locked:
             self._locked = False
         else:
             raise ValueError("Not locked")
 
-    def _check_lock(self):
+    def _check_lock(self) -> Optional[bool]:
```

Comment:
Since the possible results of the function are an exception being raised or `True`, the annotation should be `Literal[True]`.